framework/material/Form.kt

```diff
@@ -169,10 +169,16 @@ object Form {
     }
 
     @Composable
-    fun Field(label: String, fieldInput: @Composable RowScope.() -> Unit) {
-        Row(verticalAlignment = Alignment.CenterVertically) {
-            Text(value = label, modifier = LABEL_MODIFIER)
-            Row(modifier = INPUT_MODIFIER, horizontalArrangement = Arrangement.spacedBy(INNER_SPACING)) { fieldInput() }
+    fun Field(label: String, fieldHeight: Dp = FIELD_HEIGHT, fieldInput: @Composable RowScope.() -> Unit) {
+        Row {
+            Column(modifier = LABEL_MODIFIER) {
+                Row(verticalAlignment = Alignment.CenterVertically, modifier = Modifier.height(FIELD_HEIGHT)) {
+                    Text(value = label)
+                }
+            }
+            Column(modifier = INPUT_MODIFIER) {
+                Row(modifier = Modifier.height(fieldHeight)) { fieldInput() }
+            }
```

Comment:
Our fields are rows divided into two colums: labels and input fields. Now that we have variable input fields, we want labels to be left-aligned and vertically centered within a height of one 'field'. 

This behaves the same for the existing fields and as below for larger fields:
<img width="912" alt="image" src="https://user-images.githubusercontent.com/51956016/197571303-e814c447-c980-4f48-827d-7aa7aa49be3e.png">